source/app/models/project.rb

```diff
@@ -0,0 +1,11 @@
+class Project < ApplicationRecord
+  # Validations
+  validates :title, presence: true
+  validates :team_id, presence: true
+  validates :due_date, presence: true
+  validates :instructor_id, presence: true
```

Comment:
Next time you can put the validations all on the same line, ex. `validates :title, :team_id, :due_date, :instructor_id, presence: true`, but that's perf for now